lua/default-config.lua

```diff
@@ -712,14 +712,13 @@ lvim.lang = {
       },
     },
   },
-  -- R -e 'install.packages("formatR",repos = "http://cran.us.r-project.org")'
-  -- R -e 'install.packages("readr",repos = "http://cran.us.r-project.org")'
+  -- R -e 'install::packages("languageserver")'
   r = {
     formatter = {
-      exe = "format_r",
+      exe = "styler",
       args = {},
     },
-    linters = {},
+    linters = "lintr",
```

Comment:
also this should be a table `{ "lintr" }`

Reply:
Should be fixed now. Thanks

Reply:
I just squashed the commits. It's done on my side.